README.md

```diff
@@ -5,25 +5,37 @@
 Package kayvee provides methods to output human and machine parseable strings,
 with a "json" format.
 
-## [Logger Documentation](./logger)
+## [Logger API Documentation](./logger)
 
```

Comment:
Possibly add a link here to the interface docs for the logger (https://github.com/Clever/kayvee-go/tree/master/logger)